genref/go.mod

```diff
@@ -3,31 +3,29 @@ module github.com/kubernetes-sigs/reference-docs/genref
 go 1.16
 
 require (
-	github.com/google/go-cmp v0.5.2 // indirect
+	github.com/google/gofuzz v1.1.0
 	github.com/kr/text v0.2.0 // indirect
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/tengqm/kubeconfig v0.0.0-20210703133455-61479ab64f9d
-	github.com/yuin/goldmark v1.2.1
-	github.com/yuin/goldmark-highlighting v0.0.0-20200307114337-60d527fdb691
-	golang.org/x/mod v0.3.1-0.20200828183125-ce943fd02449 // indirect
-	golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073 // indirect
-	golang.org/x/tools v0.1.0 // indirect
-	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
+	github.com/tengqm/kubeconfig v0.0.0-20210707141017-7849c7b17290
```

Comment:
How come this mod is added?

Reply:
This repo was created for kubeadm config which is still embedded in the k/k repo. For generating reference for kubeadm config, we had to clone the source, eliminate its dependencies on k/k.

Reply:
👍